programs/mango-v4/src/instructions/account_buyback_fees_with_mngo.rs

```diff
@@ -55,9 +54,9 @@ pub fn account_buyback_fees_with_mngo(
     let (account_mngo_token_position, account_mngo_raw_token_index, _) =
         account.ensure_token_position(mngo_bank.token_index)?;
     let account_mngo_native = account_mngo_token_position.native(&mngo_bank);
-    if account_mngo_native.is_negative() {
+    if account_mngo_native <= I80F48::ZERO {
         msg!(
-            "account mngo token position ({} native mngo) has borrows, nothing will be bought back",
+            "account mngo token position ({} native mngo) is <= 0, nothing will be bought back",
```

Comment:
Gah, I realized we have a problem here now. If account_mngo_native == 0, then it could be that this is a newly created mango token position on `account`. When we `return Ok(())` that gets committed. So now the user has a active but zero position. 

Maybe we do something like
```
let account_mngo_native = account.token_position(mngo_bank.token_index)
    .map(|tp| tp.native(&mngo_bank))
    .unwrap_or(I80F48::ZERO);
```
so we have no pending changes when the early-return happens?